Formula/b/berkeley-db.rb

```diff
@@ -36,6 +36,9 @@ class BerkeleyDb < Formula
   end
 
   def install
+    # Work around undefined NULL causing incorrect detection of thread local storage class
+    ENV.append "CFLAGS", "-include stddef.h" if DevelopmentTools.clang_build_version >= 1600
```

Comment:
```suggestion
    ENV.append_to_cflags "-include stddef.h" if DevelopmentTools.clang_build_version >= 1600
```
Maybe?

Reply:
I was thinking of reducing scope to C code (since C++ should instead use `cstddef`). Shouldn't matter though unless lack of namespace causes collisions.